finalPrep/annotations.js

```diff
@@ -18,25 +25,35 @@ function buildABear(name, age, fur, clothes, specialPower) {
   return builtBear
 }
 
+// call on function with following arguments; name = string, age = integer, fur = string, clothes = array, special power = string
+
 buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
 buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');
 
 
-
+// create a function called fizzBuzz with the following arguments (num1, num2, range)
 //FizzBuzz
 function fizzBuzz(num1, num2, range) {
+  // make a for loop to print out the following words; "fizzbuzz", "fizz", "buzz"
   for (var i = 0; i <= range; i++) {
+    // print fizzbuzz if the current iteration divided by num1 and num2 is equal to 0 without a remainder
+
     if (i % num1 === 0 && i % num2 === 0) {
       console.log('fizzbuzz');
     } else if (i % num1 === 0) {
+      // print fizz if the current iteration is divided by num1 and result is equal to 0
       console.log('fizz');
     } else if (i % num2 === 0) {
+      // print buzz if the current iteration is divided by num2 without a remainder and is equal to 0
       console.log('buzz');
     } else {
       console.log(i);
     }
   }
 }
 
+// Invoke the function fizzBuzz twice
 fizzBuzz(3, 5, 100);
-fizzbuzz(5, 8, 400);
\ No newline at end of file
+fizzbuzz(5, 8, 400);
```

Comment:
Nice job with your annotations! Sometimes, you may want to go line by line. You've explained what is going on well, but with more complex code, there may be even more to track.

Reply:
Thank you! I'm still figuring out how to effectively annotate, and I will work on how to concisely say what I believe is going on in EACH line of code, rather than clumping together an explanation for a few lines.